client/src/components/Ticketing/ticketingmanager/TicketTypes/tickettypes.tsx

```diff
@@ -0,0 +1,118 @@
+
+import React, {useEffect} from 'react';
+import {DataGrid} from '@mui/x-data-grid';
+import Chip from '@mui/material/Chip';
+import {useAuth0} from '@auth0/auth0-react';
+
+// Web page that manages ticket types
+const TicketTypes = () => {
+  // const [ticketTypes, setTicketTypes] = useState([]);
+  const {getAccessTokenSilently} = useAuth0();
+
+
+  // Defines the columns of the grid
+  const columns = [
+    {field: 'type', headerName: 'Ticket Type', width: 300},
+    {field: 'price', headerName: 'Price', width: 150},
+    {
+      field: 'edit',
+      headerName: 'Edit',
+      sortable: false,
+      width: 100,
+      renderCell: () => {
+        return (
+          <Chip
+            label="Edit"
+            onClick={handleEditClick}
+          />
+        );
+      },
+    },
+    {
+      field: 'delete',
+      headerName: 'Delete',
+      sortable: false,
+      width: 100,
+      renderCell: () => {
+        return (
+          <Chip
+            label="Delete"
+            color="error"
+            onClick={handleDeleteClick}
+          />
+        );
+      },
+    },
+  ];
+
+  // handles the click event of the edit button
+  const handleEditClick = () => {
+    console.log('Edit Clicked');
+  };
+
+  // handles the click event of the delete button
+  const handleDeleteClick = () => {
+    console.log('Delete Clicked');
+  };
+
+
+  // Fetches all the ticket types from the API in the backend
+  const getTicketTypes = async () => {
+    const token = await getAccessTokenSilently({
+      audience: 'https://localhost:8000',
+      scope: 'admin',
+    });
+
+    try {
+      const response = await fetch(process.env.REACT_APP_ROOT_URL +
+        '/api/tickets/types', {headers: {Authorization: `Bearer ${token}`}},
+      );
+
+      console.log('Access token --> ' + process.env.REACT_APP_ROOT_URL);
+      const jsonRes = await response.json();
+      const jsonData = jsonRes.data;
+
+      console.log(jsonData);
+
+      // setTicketTypes(jsonData);
```

Comment:
Also here

Reply:
Same, since not using it right now, wanted no unused variables warning when running project